persistence-test/src/main/java/io/stargate/it/PersistenceTest.java

```diff
@@ -137,7 +136,6 @@
 
   protected abstract Persistence persistence();
 
-  @BeforeEach
```

Comment:
I had to make this run after `createPersistence()` in CassandraPersistenceIT as I had annotated that method with `@BeforeEach`. Otherwise, I would get a NPE in `new PersistenceDataStoreFactory(persistence())`.